azure/azure-release-template.yml

```diff
@@ -178,3 +178,4 @@ extends:
           pathToPublish: '$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)/pact/prescriptions-$(APIGEE_ENVIRONMENT).txt'
           artifactName: 'smoke-tests'
           publishLocation: 'Container'
+        condition: eq(variables['run_smoke_tests'], 'true')
```

Comment:
Could make this conditional on `succeeded()` as well but I thought it might be useful to see the output even if the smoke tests fail

Reply:
sounds good yea